tools/crm_mon.c

```diff
@@ -565,6 +566,10 @@ static GOptionEntry display_entries[] = {
       INDENT "See `Output Control` help for more information.",
       "SECTION(s)" },
 
+    { "only-node", 0, 0, G_OPTION_ARG_STRING, &options.only_node,
+      "Only display information regarding the given node or tag",
```

Comment:
I'm thinking just "--node" to be consistent with other tools. Too bad '-N' is already taken. Also the description is a bit tricky, I'm thinking maybe: "When displaying information about nodes, show only what's related to the given node, or to all nodes tagged with the given tag". That makes it more clear that this option is limiting various displays rather than turning on some particular display.